controllers/policies/policyserver_controller.go

```diff
@@ -46,6 +47,7 @@ type PolicyServerReconciler struct {
 //+kubebuilder:rbac:groups=policies.kubewarden.io,resources=policyservers,verbs=get;list;watch;create;update;patch;delete
 //+kubebuilder:rbac:groups=policies.kubewarden.io,resources=policyservers/status,verbs=get;update;patch
 //+kubebuilder:rbac:groups=policies.kubewarden.io,resources=policyservers/finalizers,verbs=update
+//+kubebuilder:rbac:groups=core,resources=secrets;configmaps,verbs=get;list;watch;create;update;patch
 
```

Comment:
Can we add patch and watch permissions for mutating and validating webhooks? https://github.com/kubewarden/helm-charts/pull/29/commits/e6e8a67d374f4ffaa0d412a61e7c4eda5110cd27
then I think we have the same rbac in the helm chart and in our local env

Reply:
That sounds like a great idea!
Thanks for checking with the Helm chart. Do'h, when I checked, I checked against main and not new-architecture. I was super puzzled..

Reply:
Done in https://github.com/viccuad/kubewarden-controller/commit/ce1df17001656bb972c9c4a943958a86e0b08435.

Reply:
great! thanks!